README.md

```diff
@@ -42,14 +51,26 @@ instance of the adage-server.
    ```
    Give it a few moments to start up.
 
-4. If you want to load the default Pseudomonas data into the server database,
+4. (Optional) Loading Adage models into your new instance.
```

Comment:
This is really minor, it should be `ADAGE` but not `Adage`.

Reply:
Ah, true 👍